sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveSparkSubmitSuite.scala

```diff
@@ -339,7 +339,13 @@ class HiveSparkSubmitSuite
   private def runSparkSubmit(args: Seq[String]): Unit = {
     val sparkHome = sys.props.getOrElse("spark.test.home", fail("spark.test.home is not set!"))
     val history = ArrayBuffer.empty[String]
-    val commands = Seq("./bin/spark-submit") ++ args
+    val sparkSubmit = if (Utils.isWindows) {
+      // On Windows, `ProcessBuilder.directory` does not change the current working directory.
+      new File("..\\..\\bin\\spark-submit.cmd").getAbsolutePath
+    } else {
+      "./bin/spark-submit"
```

Comment:
I really don't get why/how using the full path (possibly) makes `HiveSparkSubmitSuite` flaky because `SparkSubmitSuite` uses the full path too.

I just used the same way used in the original path just simply because 5 out of 18 builds were failed in `HiveSparkSubmitSuite`. I can't think of a workaround for Windows in this case.